source/platforms/gitlab/_tests/_gitlab_api.test.ts

```diff
@@ -6,7 +6,7 @@ import { readFileSync } from "fs"
 const nockBack = nock.back
 nockBack.fixtures = __dirname + "/fixtures"
 
-// We're testing https://gitlab.com/gitlab-org/gitlab-ce/merge_requests/27117
```

Comment:
This change dories me a bit, because this link used to work - but now it doesn't?

Reply:
The repo was moved to gitlab-foss. New MR link: https://gitlab.com/gitlab-org/gitlab-foss/-/merge_requests/27117
I updated the tests and fixtures for all Gitlab tests in https://github.com/danger/danger-js/pull/1002

Reply:
Also I think tests should run for both projectId and project path

Reply:
the link change should be reverted because https://gitlab.com/456/merge_requests/27117 is 404
or updated to the redirected link.

this is just a comment, the code can continue use numeric id